src/window/building/figures.c

```diff
@@ -30,6 +30,7 @@
 #include "translation/translation.h"
 #include "widget/city.h"
 #include "window/city.h"
+#include "utility.h"
```

Comment:
```suggestion
#include "building/utility.h"
```

For clarity reasons, we always include the folder for headers that are not this files direct header.

Also, since we order headers alphabetically, this `#include` needs to move up.